test/utils/yaml.test.ts

```diff
@@ -12,6 +11,7 @@ import {
   isPlayParam,
   isRoleParam,
   isTaskParam,
+  parseAllDocuments,
 } from '../../src/utils/yaml';
```

Comment:
I am quite curious why this was needed. I kinda find the first import more natural than the one with path. I also see other imports that we do in similar manner, including yaml/types, which likely is part of the same package. Why one works and one does not?

Is this caused by a bug on their packaging?